modules/insights-operator-new-pull-secret-enable.adoc

```diff
@@ -15,7 +15,8 @@ You can modify your existing global cluster pull secret to enable remote health
 
 .Procedure
 
-. Navigate to link:https://console.redhat.com/openshift/downloads[https://console.redhat.com/openshift/downloads].
+. Go to the link:https://console.redhat.com/openshift/downloads[Downloads] page on the {hybrid-console}.
+
 . From *Tokens* -> *Pull Secret*, click *Download*.
 +
 The file `pull-secret.txt` containing your `cloud.openshift.com` access token in JSON format downloads:
```

Comment:
checked in linux OS, the file name is `pull-secret`, does not have .txt extension